src/settings-ui/Settings.UI/Views/PowerAccentPage.xaml

```diff
@@ -23,6 +23,29 @@
                     </controls:Setting.ActionContent>
                 </controls:Setting>
 
+                <controls:SettingsGroup x:Uid="QuickAccent_Langage" IsEnabled="{Binding Mode=OneWay, Path=IsEnabled}">
+                    <controls:Setting x:Uid="QuickAccent_SelectedLangage" Icon="&#xEC12;">
```

Comment:
This icon is called "ToggleBorder" and is not appropriate for the context. Maybe `f2b7` "LocaleLanguage" is better.
https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font

Reply:
Changed that, thanks!